### ISSUE TITLE
I don't have access to certain keyboard shortcuts and intelligent help.

### ISSUE THREAD

**Issue by NolosGade** [microsoft/vscode#153847]
Issue Type: <b>Bug</b>

Bonjour,

Depuis que j'essaie de coder sur Visual Studio Code avec la version Unity 2021.2.15f1, je rencontre pas mal d'erreurs liées aux aides intelligentes à l'intérieur de Visual Studio Code. En effet, je n'ai pas accès au raccourci clavier ctrl+maj+espace qui est sensé fournir la possibilité d'avoir des conseils de développement sur des fonctions (Raycast...). Je n'ai pas non plus accès aux "ampoules" qui apparaissent en début de ligne quand une erreur peut être présente. J'ai aussi un problème avec les extensions, où celles qui sont sensés me fournir plus de raccourcis pour Unity ne fonctionnent pas (alors qu'elles sont activées globalement).

Bien à vous.

VS Code version: Code 1.68.1 (30d9c6cd9483b2cc586687151bcbcd635f373630, 2022-06-14T12:48:58.283Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz (12 x 2592)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.91GB (6.30GB free)|
|Process Argv|C:\\Users\\geeko\\Documents\\Coding Forma Travail\\RPG master\\RPG Second Attempt -g C:\\Users\\geeko\\Documents\\Coding Forma Travail\\RPG master\\RPG Second Attempt\\Assets\\Scripts\\FollowCamera.cs:1:0 --crash-reporter-id a34ec856-7050-4a61-98a6-b947335cd439|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (29)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|3.0.0
namespace|adr|1.1.2
shaderlabvscodefree|aml|1.3.0
gitlens|eam|12.1.1
eppz-code|epp|1.2.52
unity-dev-pack|fab|2.5.0
vscode-solution-explorer|fer|0.4.7
auto-using|Fud|0.7.15
docomment|k--|0.1.31
unity-code-snippets|kle|1.3.0
vscode-language-pack-fr|MS-|1.68.6150916
csharp|ms-|1.25.0
python|ms-|2022.8.1
vscode-pylance|ms-|2022.6.30
jupyter|ms-|2022.5.1001601848
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.8
vsliveshare|ms-|1.0.5625
unity-toolbox|pix|1.0.6
material-icon-theme|PKi|4.18.1
vscode-unitymeta|PTD|0.0.7
explorer-exclude-vscode-extension|Red|1.2.0
c-sharp-utilities|rev|1.2.0
LiveServer|rit|5.7.5
shader|sle|1.1.5
code-spell-checker|str|2.2.5
unity-tools|Tob|1.2.12
unity-debug|Uni|3.0.2
change-case|wma|1.0.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vscscmwlcmt:30465135
cppdebug:30492333
vsclangdf:30486550

```

</details>

**Comment by NolosGade** [comment]
English Translation (if necessary)
Hello,

Ever Since I've began coding on Visual Studio Code with Unity version 2021.2.15f1, I've encountered a lot of issues related to intelligent help inside Visual Studio Code. I don't have access to the keyboard shortcut ctrl+shift+space which is supposed to provide development tips on functions (Raycast, e.g.). I also don't have access to "light bulbs" that appear at the beginning of the line when an error may be present. I also have a problem with the extensions, where the ones that are supposed to give me more shortcuts for Unity don't work (even though they are globally enabled).

Kind regards.
